script.js

```diff
@@ -0,0 +1,384 @@
+    }
+}
+
+console.log(evenOrOdd(53));
+
+// 15. Create a function which answers the question "Are you playing banjo?".
+// If your name starts with the letter "R" or lower case "r", you are playing banjo!
+
+function areYouPlayingBanjo(name) {
+    const letterReturn = name.charAt(0);
+    if (letterReturn === 'R' || letterReturn === 'r') {
+        return name + ' plays banjo!';
+    } else {
+        return name + ' does not play banjo!';
+    }
+}
+
+console.log(areYouPlayingBanjo("Robert"));
+
+// 16. Write function bmi that calculates body mass index (bmi = weight / height2).
+//
+// if bmi <= 18.5 return "Underweight"
+// if bmi <= 25.0 return "Normal"
+// if bmi <= 30.0 return "Overweight"
+// if bmi > 30 return "Obese"
+
+function BMI(weight, height) {
+    const bmiValue = weight / (height * height);
+    if (bmiValue <= 18.5) {
+        return 'Underweight';
+    } else if (bmiValue <= 25) {
+        return 'Normal';
+    } else if (bmiValue <= 30) {
+        return 'Overweight';
+    } else {
+        return 'Obese';
+    }
+}
+
+console.log(BMI(80, 1.80));
+
+// 17. Write a rockPaperScissors  function that returns 1 if the first player won and 2 if the second player won.
+// In case of a draw, return 0.
+
+function rockPaperScissors(firstPlayer, secondPlayer) {
+    if (firstPlayer === 'rock' && secondPlayer === 'scissors') {
+        return 1;
+    } else if (firstPlayer === 'paper' && secondPlayer === 'rock') {
+        return 1;
+    } else if (firstPlayer === 'scissors' && secondPlayer === 'paper') {
+        return 1;
+    } else if (firstPlayer === 'rock' && secondPlayer === 'paper') {
+        return 2;
+    } else if (firstPlayer === 'scissors' && secondPlayer === 'rock') {
+        return 2;
+    } else if (firstPlayer === 'paper' && secondPlayer === 'scissors') {
+        return 2;
+    }
+    return 0;
+}
+
+console.log(rockPaperScissors('rock', 'paper'));
+
+// 18. Create the getCalculationResult function
+// it should accept three arguments:
+// the first number
+// the second number
+// the calculation type
+// the calculation type should be a string that is either '+', '-', '*', or '/'
+// the function should return the result of the calculation based on the numbers and the type of the calculation
+// if the calculation type is not recognized, return null
+
+function getCalculationResult(firstNumber, secondNumber, calculationType) {
+    const result = eval(firstNumber + calculationType + secondNumber);
+    if (calculationType === '+' || calculationType === '-' || calculationType === '*' || calculationType === '/') {
+        return result;
+    }
+    return null;
+}
+
+console.log(getCalculationResult(3, 3, '%'));
+
+// 19. Implement the getPercentageValue function
+// it should accept two arguments:
+// the base number
+// the percentage
+// the function should return the percentage value based on the base number and the percentage.
+
+function getPercentageValue (baseNumber, percentageNumber) {
```

Comment:
There is an unnecessary space here

Reply:
I don't see any unnecessary space there.

Reply:
Between `getPercentageValue` and `(baseNumber, percentageNumber)`